src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1285,12 +1285,10 @@ private void InitializeHelix()
                 Name = HeadLightName
             };
 
-            //TODO verify this binding is working.
             headLight.SetBinding(
                 DirectionalLight3D.DirectionProperty, 
-                new Binding("LookDirection") {
-                    Source = this,
-                    Path = new PropertyPath("Camera.LookDirection")
+                new Binding(nameof(PerspectiveCamera.LookDirection)) {
```

Comment:
this is the important change.

Reply:
Would you put comments why it is important? Not setting **Path** or?